index.html

```diff
@@ -1,657 +1,1338 @@
+                                <a id="button-9752993433" class="cmp-button" href="/content/leaderimpact/lang/en2/start-a-group.html">
+                                  <span class="cmp-button__text">Start a Group</span>
+                                </a>
+                              </div>
+  
+                              <div class="responsivegrid aem-GridColumn aem-GridColumn--default--12 layout-grid navigation-wrap">
+                                <div id="container-d76502f407" class="cmp-container">
+                                  <div class="aem-Grid aem-Grid--12 aem-Grid--default--12 nav-content-wrap">
+                                    <div class="navigation nav-wrap aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                                      <div id="container-93b0189817" class="cmp-container">
+                                        <nav id="navigation-9706c9855f" class="cmp-navigation" itemscope="" itemtype="http://schema.org/SiteNavigationElement">
+                                          <ul class="cmp-navigation__group">
+                                            <li class="cmp-navigation__item cmp-navigation__item--level-0">
+                                              <a href="/content/leaderimpact/lang/en2/join-a-group.html" class="cmp-navigation__item-link">Join a Group</a>
+                                            </li>
+                                            <li class="cmp-navigation__item cmp-navigation__item--level-0">
+                                              <a href="/content/leaderimpact/lang/en2/start-a-group.html" class="cmp-navigation__item-link">Start a Group</a>
+                                            </li>
+                                            <li class="cmp-navigation__item cmp-navigation__item--level-0">
+                                              <a href="/content/leaderimpact/lang/en2/resources.html" class="cmp-navigation__item-link">Resources</a>
+                                            </li>
+                                            <li class="cmp-navigation__item cmp-navigation__item--level-0">
+                                              <a href="/content/leaderimpact/lang/en2/stories.html" class="cmp-navigation__item-link">About</a>
+                                            </li>
+                                          </ul>
+                                        </nav>
+                                      </div>
+                                    </div>
+  
+                                    <div class=" responsivegrid aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                                      <div id="container-8ebb211cec" class="cmp-container">
+                                        <div class="aem-Grid aem-Grid--12 aem-Grid--default--12 social-wrap ">
+                                          <div class="image aem-GridColumn aem-GridColumn--default--12 layout-grid">

```

Comment:
You're missing a `<div id="container-213142dc3b" class="cmp-container">` between these two divs.

Reply:
Isn't <div id="container-213142dc3b" class="cmp-container"> supposed to go above <div class="aem-Grid aem-Grid--12 aem-Grid--default--12 social-wrap "> like its already done in the screen shot?

![image](https://user-images.githubusercontent.com/82415409/213308629-b5362558-0d47-405c-ae33-61c80e415cc7.png)

Reply:
It looks like my initial comment is incorrect. The divs are there, but there is an extra class where it shouldn't be, which was throwing me off. There are 2 structures for the container component:

The "responsive grid" type (this allows authors to change the width of the container themselves):
```
<div class="container responsivegrid customclass1 customclass2...customclassn">
  <div id="customId or generated" class="cmp-container">
    <div class="aem-Grid aem-Grid--12 aem-Grid--default--12">
      ...other components go in here
    </div>
  </div>
</div>
```
The "simple" container:
```
<div class="container responsivegrid customclass1 customclass2...customclassn">
  <div id="customId or generated" class="cmp-container">
    ...other components go in here
  </div>
</div>
```

Your screenshot looks correct (using the first type), but the code in this pull request looks like this (note the extra `social-wrap` class on line 3):
```
<div class="container responsivegrid social-wrap">
  <div id="container-8ebb211cec" class="cmp-container">
    <div class="aem-Grid aem-Grid--12 aem-Grid--default--12 social-wrap ">
      ...other components
    </div>
  </div>
</div>
```

Reply:
I have not pushed these changes yet, I was working on the contact page conversion... I'll push these by the end of the day
the screen shot which I shared was after I made the changes